### PR TITLE
Convert address to telegram handle

### PR DESCRIPTION
**Pull request by ldinghan**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/CliSyntax.java [hunk 1]
```diff
@@ -10,6 +10,7 @@ public class CliSyntax {
     public static final Prefix PREFIX_PHONE = new Prefix("p/");
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
```
Comment (Respirayson) [inline src/main/java/seedu/address/logic/parser/CliSyntax.java]: perhaps we could delete the old prefix_address

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 2]
```diff
@@ -52,7 +52,7 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
+        telegramHandle.setText(person.getTelegramHandle().value);
```
Comment (Respirayson) [inline src/main/java/seedu/address/ui/PersonCard.java]: tbh i guess we could prepend @ to the front of the value just so on the ui it looks like an actual tele handle

**Review by Respirayson — APPROVED** [review]
few nits but overall LGTM

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 2 above)
Comment (ldinghan) [reply src/main/java/seedu/address/ui/PersonCard.java]: sounds like a good solution

File: src/main/java/seedu/address/logic/parser/CliSyntax.java (hunk 1 above)
Comment (ldinghan) [reply src/main/java/seedu/address/logic/parser/CliSyntax.java]: good catch!

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 3]
```diff
@@ -52,7 +52,7 @@ public PersonCard(Person person, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
         phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
+        telegramHandle.setText(String.format("@%s", person.getTelegramHandle().value));
```
Comment (raydenlim) [inline src/main/java/seedu/address/ui/PersonCard.java:55]: Codecov angry

File: src/main/java/seedu/address/model/person/TelegramHandle.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+/**
+ * Represents a Telegram handle of a person.
+ * A Telegram handle can only consist of lowercase and uppercase letters, digits, and underscores.
+ * Usernames are case-insensitive.
+ */
+public class TelegramHandle {
+    public static final String MESSAGE_CONSTRAINTS = "Telegram Handles can only use a-z, 0-9 and underscores. "
+            + "Usernames are case-insensitive";
+
+    public static final String VALIDATION_REGEX = "^[a-zA-Z0-9_]{1,}$";
```
Comment (raydenlim) [inline src/main/java/seedu/address/model/person/TelegramHandle.java:15]: Might  want to add the @ for telegram, 
OR append it in the Personcard

Comment (raydenlim) [reply src/main/java/seedu/address/model/person/TelegramHandle.java:15]: then check if it exist or some logic to prevent 2 @ from printing e.g @@l_dinghan

**Review by raydenlim — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/ui/PersonCard.java (hunk 3 above)
Comment (ldinghan) [reply src/main/java/seedu/address/ui/PersonCard.java:55]: Yes unfortunately we don't test ui here ;p

File: src/main/java/seedu/address/model/person/TelegramHandle.java (hunk 4 above)
Comment (ldinghan) [reply src/main/java/seedu/address/model/person/TelegramHandle.java:15]: appended into PersonCard

**Pull request merged**